provider/cloudflare/cloudflare.go

```diff
@@ -410,23 +416,43 @@ func (p *CloudFlareProvider) ApplyChanges(ctx context.Context, changes *plan.Cha
 		add, remove, leave := provider.Difference(current.Targets, desired.Targets)
 
 		for _, a := range remove {
-			cloudflareChanges = append(cloudflareChanges, p.newCloudFlareChange(cloudFlareDelete, current, a, current))
+			change, err := p.newCloudFlareChange(cloudFlareDelete, current, a, current)
```

Comment:
not covered with unit test
![Screenshot 2025-06-22 at 10 06 37](https://github.com/user-attachments/assets/0f2ebb26-b30c-4942-8b14-b3c081cf1f54)